2019/el-gr/src/0xd1-acknowledgments.md

```diff
@@ -0,0 +1,42 @@
+Ευχαριστίες
+===============
+
+## Ευχαριστίες προς τους Συντελεστές
+
+Θα θέλαμε να ευχαριστήσουμε τους ακόλουθους συντελεστές που συνεισέφεραν δημόσια στο GitHub ή με άλλα μέσα:
+
+* 007divyachawla
+* Abid Khan
+* Adam Fisher
+* anotherik
+* bkimminich
+* caseysoftware
+* Chris Westphal
+* dsopas
+* DSotnikov
+* emilva
+* ErezYalon
+* flascelles
+* Guillaume Benats
+* IgorSasovets
+* Inonshk
+* JonnySchnittger
+* jmanico
+* jmdx
+* Keith Casey
+* kozmic
+* LauraRosePorter
+* Matthieu Estrade
+* nathanawmk
+* PauloASilva
+* pentagramz
+* philippederyck
+* pleothaud
+* r00ter
+* Raj kumar
+* Sagar Popat
+* Stephen Gates
+* thomaskonrad
+* xycloops123
+* Athanasios Emmanouilidis
+* Apostolos Giannakidis
```

Comment:
This list should remain synchronized with the master version as well as other translations.
The names in this list correspond to those that helped putting together the 2019 version.

Translators are acknowledge [here](https://owasp.org/www-project-api-security/#div-translations).
Additionally, you can add a "Translation Notes" sub-section in the "Release Notes" section as [the one you find in the Portuguese translation](https://github.com/OWASP/API-Security/blob/master/2019/pt-pt/src/0x04-release-notes.md#notas-da-tradu%C3%A7%C3%A3o).